containers/FeedList/Feed.ts

```diff
@@ -0,0 +1,59 @@
+export class Feed {
+    id: number
+    title: string
+    userImage: string
+    userName: string
+    createDate: string
+    tags: string[]
+    code: string
+    viewCount: number
+    likeCount: number
+    commentCount: number
+    isLiked: boolean
+    isBookmarked: boolean
+
+    constructor(id: number, title: string, userImage: string, userName: string, createDate: string,
+                tags: string[], code: string, viewCount: number, likeCount: number, commentCount: number,
+                isLiked: boolean, isBookmarked: boolean) {
+        this.id = id;
+        this.title = title;
+        this.userImage = userImage;
+        this.userName = userName;
+        this.createDate = createDate;
+        this.tags = tags;
+        this.code = code;
+        this.viewCount = viewCount;
+        this.likeCount = likeCount;
+        this.commentCount = commentCount;
+        this.isLiked = isLiked;
+        this.isBookmarked = isBookmarked;
+    }
+}
```

Comment:
```suggestion

    constructor(
       public id: number,
       public title: string,
       public userImage: string,
       public userName: string,
       public createDate: string,
       public tags: string[],
       public code: string,
       public viewCount: number, 
       public likeCount: number,
       public commentCount: number,
       public isLiked: boolean,
       public isBookmarked: boolean) { }
}
```
외부에서 접근 가능성에 따라 public or private 붙여서 쓰면 this. 에 대입하지 않아도 자동으로 초기화 됩니다.